colpali_engine/models/idefics_2/colidefics_2/processing_colidefics_2.py

```diff
@@ -18,23 +18,35 @@ def __init__(self, *args, **kwargs):
     def process_images(
         self,
         images: List[Image.Image],
+        contexts_prompts: Optional[List[str]] = None,
     ) -> BatchEncoding:
         """
         Process images for ColIdefics2.
         """
         texts_doc: List[str] = []
         images = [image.convert("RGB") for image in images]
 
-        for _ in images:
-            messages_doc = [
-                {
-                    "role": "user",
-                    "content": [
-                        {"type": "text", "text": "Describe the image."},
-                        {"type": "image"},
-                    ],
-                },
-            ]
+        for i, _ in enumerate(images):
```

Comment:
I would revert the changes for this model and deprecate, but other optoin is to keep it but be consistyent with idefics3 comment I did